src/components/DefineCompany/BusinessContactInfo.vue

```diff
@@ -1,16 +1,23 @@
 <template>
   <div>
     <div v-if="!isEditing">
-      <div>Email Address: {{ !!contact.email ? contact.email : 'Not entered'}}</div>
-      <div>Phone: {{ !!contact.phone ? contact.phone : 'Not entered' }}
+      <div id="lbl-email">Email Address: {{ !!contact.email ? contact.email : 'Not entered'}}</div>
```

Comment:
Just curious, was there a standard set for what we should be using as id names or did you come up with this? Really just asking for my own knowledge :)

Reply:
No No :) No standards at all.. i just added it like that